pilot/pkg/serviceregistry/kube/controller/ambientindex.go

```diff
@@ -81,7 +82,7 @@ func (a *AmbientIndex) Lookup(ip string) []*model.AddressInfo {
 			},
 		}
 
-		return []*model.AddressInfo{&model.AddressInfo{Address: addr}}
+		return []*model.AddressInfo{{Address: addr}}
 	}
```

Comment:
Can you add more comments in the code - what is the last case with services[ip] ? Any service we may want to exclude in the future ( for example with affinity, per-node, etc - where simply load balancing will break ) ?